### ISSUE TITLE
Newtonsoft.Json.JsonConvert.DeserializeObject is not working in iOS15 when application is killed

### ISSUE THREAD

**Issue by maheshkumar1989** [JamesNK/Newtonsoft.Json#2631]
Newtonsoft.Json.JsonConvert.DeserializeObject<classname>(json); This code is not working when application is killed and this code is execute on the click of notification Approve button from Notification Center. Same code is working fine when the application is in background mode. Newtonsoft.Json.JsonConvert.DeserializeObject is not working in iOS15  while working fine with iOS14.  

Any help Appreciated!
Thanks

**Comment by bartelink** [comment]
Best to write a question on stack overflow showing details of your code and the exact messages and/or stack traces and what you know of the running environment - this issue list gets far fewer eyes than that and is hence best used only for cases where you've established a lot more details as to what the problem may be

**Comment by maheshkumar1989** [comment]
Okay, Thanks @bartelink for the update

**Comment by bartelink** [comment]
If you do write such a SO post, paste a link here and close this to complete the circle ;)

**Comment by maheshkumar1989** [comment]
SO Post Link
https://stackoverflow.com/questions/70540587/newtonsoft-json-jsonconvert-deserializeobject-is-not-working-in-ios15-when-appli

**Comment by maheshkumar1989** [comment]
This is issue not related to the Newtonsoft.Json library and the code(Newtonsoft.Json.JsonConvert.DeserializeObject(json);)  working fine. I observed when the user force quit or killed  the iOS application by swiping up, after force quit no code is execute in the background. What I did, I implemented a loop of 50000 counts and write the count in the log file. First killed the application then send a silent notification and study the logs in the log file and try the same 10-15 times and what I observed every time counts in the log file was different in the log file. Sometime count was 0-50, 0-158,0-999,0-8000,0-20,0-18 and 2 or 3 times count not started.
